src/App.js

```diff
@@ -7,10 +7,15 @@ import UserPage from './pages/User/UserPage';
 import TaskPage from './pages/TaskPage';
 import Stylesheets from './pages/Stylesheets/Stylesheets';
 import NasaPicsPage from './pages/NasaPics/NasaPicsPage';
+import { useStyles } from './styles';
+import DarkThemeProvider from './DarkThemeProvider';
 
-function App() {
+import { Paper } from '@material-ui/core';
+
+const Router = () => {
+    const classes = useStyles();
     return (
-        <HashRouter>
+        <Paper square className={classes.paper}>
```

Comment:
why paper?

Reply:
Becouse it works good as a background for dark theme, not sure if there any other components for it.